### PR TITLE
docs: cli option help additions

### PR DESCRIPTION
**Pull request by thephez**
Adds help text for some options that were lacking it

### CODE REVIEW

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

File: src/init.cpp [hunk 1]
```diff
@@ -598,6 +598,12 @@ std::string HelpMessage(HelpMessageMode mode)
...
+    strUsage += HelpMessageOpt("-highsubsidyblocks=<n>", strprintf(_("The number of blocks with a higher than normal subsidy to mine at the start of a devnet (default: 0)")));
+    strUsage += HelpMessageOpt("-highsubsidyfactor=<n>", strprintf(_("The factor to multiply the normal block subsidy by while in the highsubsidyblocks window of a devnet (default: 1)")));
+    strUsage += HelpMessageOpt("-llmqchainlocks=<quorum name>", strprintf(_("Override the default LLMQ type used for ChainLocks on a devnet. Allows using ChainLocks with smaller LLMQs. (default: llmq50_60)")));
+    strUsage += HelpMessageOpt("-llmqdevnetparams=<size:threshold>", strprintf(_("Override the default LLMQ size for the LLMQ_DEVNET quorum (default: 10:6)")));
+    strUsage += HelpMessageOpt("-llmqinstantsend=<quorum name>", strprintf(_("Override the default LLMQ type used for InstantSend on a devnet. Allows using InstantSend with smaller LLMQs. (default: llmq50_60)")));
+    strUsage += HelpMessageOpt("-llmqtestparams=<size:threshold>", strprintf(_("Override the default LLMQ size for the LLMQ_TEST quorum (default: 3:2)")));
```
Comment (xdustinface) [inline src/init.cpp:606]:
```suggestion
    strUsage += HelpMessageOpt("-highsubsidyblocks=<n>", _("The number of blocks with a higher than normal subsidy to mine at the start of a devnet (default: 0)"));
    strUsage += HelpMessageOpt("-highsubsidyfactor=<n>", _("The factor to multiply the normal block subsidy by while in the highsubsidyblocks window of a devnet (default: 1)"));
    strUsage += HelpMessageOpt("-llmqchainlocks=<quorum name>", _("Override the default LLMQ type used for ChainLocks on a devnet. Allows using ChainLocks with smaller LLMQs. (default: llmq50_60)"));
    strUsage += HelpMessageOpt("-llmqdevnetparams=<size:threshold>", _("Override the default LLMQ size for the LLMQ_DEVNET quorum (default: 10:6)"));
    strUsage += HelpMessageOpt("-llmqinstantsend=<quorum name>", _("Override the default LLMQ type used for InstantSend on a devnet. Allows using InstantSend with smaller LLMQs. (default: llmq50_60)"));
    strUsage += HelpMessageOpt("-llmqtestparams=<size:threshold>", _("Override the default LLMQ size for the LLMQ_TEST quorum (default: 3:2)"));
```

File: src/init.cpp [hunk 2]
```diff
@@ -610,6 +616,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-logtimestamps", strprintf(_("Prepend debug output with timestamp (default: %u)"), DEFAULT_LOGTIMESTAMPS));
     strUsage += HelpMessageOpt("-maxtxfee=<amt>", strprintf(_("Maximum total fees (in %s) to use in a single wallet transaction or raw transaction; setting this too low may abort large transactions (default: %s)"),
         CURRENCY_UNIT, FormatMoney(DEFAULT_TRANSACTION_MAXFEE)));
+    strUsage += HelpMessageOpt("-minimumdifficultyblocks=<n>", strprintf(_("The number of blocks that can be mined with the minimum difficulty at the start of a devnet (default: 0)")));
```
Comment (xdustinface) [inline src/init.cpp:619]:
```suggestion
    strUsage += HelpMessageOpt("-minimumdifficultyblocks=<n>", _("The number of blocks that can be mined with the minimum difficulty at the start of a devnet (default: 0)"));
```

File: src/init.cpp [hunk 3]
```diff
@@ -619,10 +626,13 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-printtodebuglog", strprintf(_("Send trace/debug info to debug.log file (default: %u)"), 1));
     strUsage += HelpMessageOpt("-shrinkdebugfile", _("Shrink debug.log file on client startup (default: 1 when no -debug)"));
     strUsage += HelpMessageOpt("-sporkaddr=<dashaddress>", strprintf(_("Override spork address. Only useful for regtest and devnet. Using this on mainnet or testnet will ban you.")));
+    strUsage += HelpMessageOpt("-sporkkey=<privatekey>", strprintf(_("Set the private key to be used for signing spork messages.")));
```
Comment (xdustinface) [inline src/init.cpp:629]:
```suggestion
    strUsage += HelpMessageOpt("-sporkkey=<privatekey>", _("Set the private key to be used for signing spork messages."));
```

**Review by xdustinface — CHANGES_REQUESTED** [review]
Looks good but all those `strprintf` calls can be dropped :)

**Review by xdustinface — APPROVED** [review]
utACK

**Review by UdjinM6 — APPROVED** [review]
re-utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Pull request merged**